src/client-common/client-telemetry-helper.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Great feedback, I wasn't aware module was a common place for clients, I've renamed the telemetry-helper and moved it to modules (will be in my next commit)